pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/InternalConfigConstants.java

```diff
@@ -36,4 +36,10 @@
   public static final String PARTITION_FUNCTION_CONFIG = "partition.function";
 
   public static final String SORTED_COLUMN_CONFIG = "sorted.column";
+  public static final String ENABLE_PARTITIONING = "enable.partitioning";
+
+  // max records per file in each partition. No effect otherwise.
+  public static final String PARTITION_MAX_RECORDS_PER_FILE = "partition.max.records.per.file";
```

Comment:
It doesn't have to be the case when partition is enabled, right?

Reply:
that is what we discussed the other day with @Jackie-Jiang as without the ability to balance files, not worth it for normal cases

Reply:
Let's be conservative for now. If we find new use cases that require this for non-partitioning case, we can add then